src/Result/InvoicePaymentMethod.php

```diff
@@ -67,4 +67,17 @@ public function getPaymentMethod(): string
         $data = $this->getData();
         return $data['paymentMethod'];
     }
+
+    public function getCryptoCode(): string
+    {
+        $data = $this->getData();
+        // For future compatibility check if cryptoCode exists.
+        if (isset($data['cryptoCode'])) {
```

Comment:
Does this mean that a new field `cryptoCode` is coming and confirmed?

Reply:
I will open an issue, there is a Use-case (as discussed with Mitchell) and it would be consistent to also have it here, imo.